src/main/java/com/softserve/service/impl/StudentServiceImpl.java

```diff
@@ -164,7 +166,7 @@ public Student delete(Student object) {
      * First line of the file is a header.
      * All subsequent lines contain data about students.
      * <p>
-     * "surname","name","patronymic","email"
+     * "surname","name","patronymic",EMAIL
```

Comment:
better to use literals in java doc rather than class variables

Reply:
I also think it will be better